src/table/virtualized-table/__tests__/Wrapper.spec.tsx

```diff
@@ -50,16 +48,7 @@ describe("<Wrapper />", () => {
 
   afterEach(() => jest.restoreAllMocks());
 
-  it("should not render table with pagination", () => {
```

Comment:
Yes but no. Since the `PaginationFooter` component is mocked and that component takes care of the decision if the table should render without pagination.

So the test case is no longer possible. Unless you render the whole footer which at the time of writing this was "too much work".

Reply:
ah yes I see you are mocking the Table component as well, then this makes sense